doc/voice-api.md

```diff
@@ -0,0 +1,51 @@
+# Almond Voice
+
+Currently, Almond Voice provides two REST endpoints that provides TTS and STT functionality for Almond-based services, both hosted at [voice.almond.stanford.edu](https://voice.almond.stanford.edu). Support for websocket-based streaming will be added in the future.
```

Comment:
We should probably note that this API is _experimental_ and likely to be moved around and/or merged with other API endpoints. I don't expect we'll keep this standalone server alive long past TreeHacks.

Reply:
Will do